packages/models/tests/cache/call.test.ts

```diff
@@ -0,0 +1,64 @@
+import { Cache } from "@/cache";
+import { expect } from "chai";
+
+const cache = new Cache(process.env.REDIS_URL || "");
```

Comment:
No to create this cache object for each test file, we can move it into a fixture.